frontend/src/citizen-frontend/generated/api-clients/attachment.ts

```diff
@@ -40,3 +47,124 @@ export function getAttachment(
     url: uri`${client.defaults.baseURL ?? ''}/citizen/attachments/${request.attachmentId}/download/${request.requestedFilename}`
   }
 }
+
+
+/**
+* Generated from fi.espoo.evaka.attachment.AttachmentsController.uploadApplicationAttachmentCitizen
+*/
+export async function uploadApplicationAttachmentCitizen(
+  request: {
+    applicationId: ApplicationId,
+    type: ApplicationAttachmentType,
+    file: File
+  },
+  options?: {
+    onUploadProgress?: (event: AxiosProgressEvent) => void
+  }
```

Comment:
Vaikuttaisi siltä ettei `onUploadProgress`:ia passata eteenpäin

Reply:
Argh, on tippunut jossain refaktoroinnissa pois :man_facepalming: